.github/workflows/stale.yml

```
@@ -0,0 +1,29 @@
+name: Close stale issues and PRs
+
+on:
+  schedule:
+    # run weekly on sunday
+    - cron: "0 0 * * 0"
+  workflow_dispatch:
+
+permissions:
+  issues: write
+  pull-requests: write
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v9
+        with:
+          days-before-issue-stale: 14
+          days-before-pr-stale: 360 # a year
```

```suggestion
          days-before-pr-stale: 30
```

This is too long. 30 could be too short as well, but I think it'd be good to consider things "stale" earlier so they're bumped and searched for separately quicker